homeassistant/components/workday/config_flow.py

```diff
@@ -67,7 +68,30 @@ def add_province_to_schema(
         ),
     }
 
-    return vol.Schema({**DATA_SCHEMA_OPT.schema, **add_schema})
+    return vol.Schema({**add_schema, **schema.schema})
+
+
+def add_language_to_schema(
+    schema: vol.Schema,
+    country: str,
+) -> vol.Schema:
+    """Update schema with a language (one of available localizations)."""
+    cls: HolidayBase = getattr(holidays, country)
+    if not cls.supported_languages:
+        return schema
+
+    language_options = [NONE_SENTINEL, *cls.supported_languages]
+    add_schema = {
+        vol.Optional(CONF_LANGUAGE, default=cls.default_language): SelectSelector(
```

Comment:
Langauge is not an integration option; it is a Home Assistant configuration.

Integrations should use the English language and use Home Assistant translations in case they want to provide translations.

../Frenck

Reply:
It's which language you would configure named holidays in (so has nothing to do how it's presented to the user).
So for myself do I want to remove christmas as holiday by writing `Christmas Eve` or `Julafton` (or any other language supported by the country).